Resources/Prototypes/_White/Actions/types.yml

```
@@ -1,3 +1,18 @@
+  description: Creates hardlight spear in your hands.
+  noSpawn: true
+  components:
+  - type: InstantAction
+    useDelay: 1.5
+    itemIconStyle: BigAction
+    priority: -20
+    icon:
+      sprite: _White/Objects/Weapons/Melee/Spear/hardlight_spear.rsi
+      state: spear
+    event: !type:ActivateSpawnImplantEvent
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)